### DISCUSSION TITLE
Add Fa icon to select option

### DISCUSSION THREAD

**Discussion by arispap2003** [orchidsoftware/platform#2388] [General]
How can I add font awsome icon to an option?

**Comment by tabuna** [comment]
You can learn about connecting icons in the following document: https://orchid.software/en/docs/icons/#custom-icons

**Comment by arispap2003** [comment]
Can you write an example calling a Select box with a  fontawesome  icon 
as the 1st option?|
|

**Comment by hoangnamitc** [comment]
How to use icon from Orchid to outside website.

eg:
icon of category.
selectbox choose icon of Orchid, call from website.

**Comment by arispap2003** [comment]
Select::make('helppost.hashtags')
                                        ->options([
                                            0  => 'What code to put here to have a FA-ICON? ',
                                            1  => '',

                                            2  => 'Option 2',
                                        ])
